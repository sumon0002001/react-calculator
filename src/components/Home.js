import React from 'react';

function Home() {
  return (
    <div className="Home">
      <h1>Welcome to our page</h1>
      <p>
        Lorem ipsum odor amet, consectetuer adipiscing elit.
        Facilisi sollicitudin purus auctor at cursus amet auctor
        lobortis velit. Fames nisi ullamcorper gravida cubilia conubia
        accumsan vitae sollicitudin. Quisque vulputate habitasse libero
        dapibus primis facilisis elementum. Quam est ex porttitor
        senectus congue magna lacinia. Morbi torquent convallis rutrum sit,
        dui cubilia quisque venenatis. Velit nullam aliquam posuere
        himenaeos felis proin magnis ultricies. Convallis malesuada aenean
        diam est quam.
      </p>
      <p>
        Erat quisque etiam consequat facilisis quisque;
        adipiscing mauris erat. Congue est dapibus sit eu mi
        non class felis ridiculus. Gravida nullam egestas suscipit
        parturient maximus malesuada. Sit viverra nascetur elementum
        tristique platea nullam molestie. Dui aptent fermentum condimentum
        efficitur eu ad phasellus nascetur.
        Litora himenaeos convallis magnis neque curae condimentum maximus.
        Fringilla dolor nulla dis ante congue nec montes ut.
      </p>
    </div>
  );
}

export default Home;
