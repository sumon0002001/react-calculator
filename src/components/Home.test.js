import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('Renders the Home page', () => {
  const home = renderer
    .create(<Home />)
    .toJSON();
  expect(home).toMatchSnapshot();
});
