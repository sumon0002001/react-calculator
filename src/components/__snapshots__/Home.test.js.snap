// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Home page 1`] = `
<div>
  <h1>
    Welcome to Our Page
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, facere sapiente. Distinctio facilis dignissimos, ipsam vero qui asperiores. Sapiente esse cupiditate saepe tempora, eum totam quibusdam voluptatem quae labore aliquid?
  </p>
  <br />
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid, facere sapiente. Distinctio facilis dignissimos, ipsam vero qui asperiores. Sapiente esse cupiditate saepe tempora, eum totam quibusdam voluptatem quae labore aliquid?
  </p>
</div>
`;
